_episodes/02-markdown.md

```diff
@@ -221,8 +221,46 @@ keypoints:
 {: .challenge }
 
 
+Linking other websites is an essential feature of a website.
+Bare URLs like https://carpentries.org/ can be made clickable by wrapping them into 
+angle brackets like this <https://carpentries.org/>.
```

Comment:
I think you might need to escape the brackets with \< and \> if you want to see them in the text instead of have them create the link.

Reply:
Thanks! I didn't think of this and tried to be more specific in 199da9be92be5f7f83abe089259750b779d6794e